lib/live_fixtures/import/fixtures.rb

```diff
@@ -62,8 +67,15 @@ def each_table_row_with_label
       join_table_rows.each do |table_name, rows|
         rows.each do |targets:, association:, label:|
           targets.each do |target|
-            assoc_fk = @label_to_id[target] || target
-            row = { association.foreign_key             => @label_to_id[label],
+            # This song and dance is so we can specify HABTM like so:
+            # dogs: 12, 32, 144 instead of dogs: dogs_12, dogs_32, dogs_144
```

Comment:
LiveFixtures works with yml files that were created by hand as well. There are no requirements on the format of the label for each record.